### PR TITLE
quay-enterprise: document set up of an ELB with Proxy Protocol for QE

### PR DESCRIPTION
**Pull request by kbrwn**
Needed to replace older documentation on Quay.io
References product correctly (Quay Enterprise)
Provides exact cli commands needed to create ELB policy
Helpful Troubleshooting section with common issues

### CODE REVIEW

**Comment by kbrwn** [comment]
@josephschorr @jzelinskie @joshix please review.

File: quay-enterprise/setup-elb.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+| -------- | :------------: | :------------: |
+| HAProxy  |       ✓        |       ✓        |
+| ELB      |       ✓        |       ✓        |
+| nginx    |       ✓        |       ✕        |
+
+## Setting up Amazon Elastic Load Balancer
+
+Setting up an ELB with Proxy Protocol enabled requires an existing classic ELB and access to the [aws cli tool](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html). AWS documentation on creating an ELB can be found [here](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-getting-started.html).
+
+ELB Name: quay-loadbalancer
```
Comment (josephschorr) [inline quay-enterprise/setup-elb.md]: Put ` around the names

File: quay-enterprise/setup-elb.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+```
+$ curl -k https://${LoadBalancer}/v1/_ping
+true
+```
+
+`docker login` and `docker push` should be operational. 
+
+## Troubleshooting 
+
+[Security group](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html) settings are often the culprit if the ELB is not resolving but the health check reports the instance as in service. The security group for the ELB should be open for inbound traffic from port `80`, `443`, and `8443`.
```
Comment (josephschorr) [inline quay-enterprise/setup-elb.md]: Add language that if the security group is misconfigured, it generally shows up as the connection hanging

File: quay-enterprise/setup-elb.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+### Add Health Check 
+
+After the listeners have been configured, the health checking endpoint needs to be configured to use the previously unused port 443. When properly configured, the Listeners tab for the ELB should appear like so:
+
+<img src="img/healthcheck.png" class="img-center" alt="Healthcheck">
+
+### Check Functionality 
+
+If the ELB with Proxy Protocol is functioning properly curling the `/v1/_ping` endpoint should return `true`. Replace ${LoadBalancer} with the A record of the quay-loadbalancer ELB..
```
Comment (josephschorr) [inline quay-enterprise/setup-elb.md]: `A` record

**Pull request merged**